functions/src/repository/pull-request.ts

```diff
@@ -0,0 +1,68 @@
+import * as firebase from 'firebase-admin';
+import { FirebaseAdmin, QuerySnapshot } from '../client/firebase-admin';
+import { GitHubClient } from '../client/github';
+import { Logger } from '../client/logger';
+
+import { BuildTimes, GitHubPullRequestStatusInput, GitHubPullRequestStatusMapper, GitHubPullRequestStatusModel } from '../mappers/github/status.mapper';
+
+/**
+ * Return the build times for all type of context for any PR
+ * @param statuses GitHubPullRequestStatusModel[]
+ */
+export const getPullRequestBuildTime: any = (statuses: GitHubPullRequestStatusModel[]): BuildTimes[] => {
```

Comment:
It is best we do not use `any`

Reply:
What should be the type of this function here ?

Reply:
yes you are right, we use have a type for this, we can leave it as `any` for now